rails_multisite.gemspec

```diff
@@ -17,9 +17,9 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = RailsMultisite::VERSION
 
-  gem.required_ruby_version = ">=2.5.0"
+  gem.required_ruby_version = ">=3.0.0"
 
   %w{activerecord railties}.each do |gem_name|
-    gem.add_dependency gem_name, "> 5.0", "< 7.1"
+    gem.add_dependency gem_name, ">= 6.0", "<= 7.1"
```

Comment:
Can we drop the upper bound (`<= 7.1`) so that we don't have to release a new version of this gem when 7.2 comes around? (and doesn't have breaking changes)

(and I'd split that into two `gem.add_dependency` lines)